Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pickable.py

```diff
@@ -51,17 +52,36 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def pickable(cls, commits):
-        filtered_commits = []
+    def pickable(cls, commits, repository, commits_story=None):
+        filtered_in = set()
```

Comment:
I'd suggest a more descriptive name for `filtered_in` here, something that indicates that these are commits that were originally authored on the branch, such as `original_authored_commit_shas`.